docs/developing/gasless.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Gasless transactions
+proofedDate: na
+iterationBy: na
+includedInSite: true
+approvedBy: na
+comment: There are code examples (React>> https://codesandbox.io/s/t3fid3) and https://thirdweb.com/neon-evm-devnet/0xF3eBc32292F4BbFB83DECB97Eb42d95da968f775/sources >> bit confused though, the flow described means that the dApp dev has nothing to do but accept the 0 cost tx. If these code examples "demonstrate how to implement gasless transactions" does that mean they apply to Proxy Operators?
+---
+
+## TL;DR
+
+- Neon EVM provides a starter pack of gasless transactions
+- Buy NEON tokens from your chosen DEX by using free NEON
+
+## Introduction
+
+What happens if you do not yet own enough (or any) NEON to pay for your user's first transactions? Don't worry, we have you covered with a starter pack of gasless transactions.
+
+These transactions are still handled by your chosen [Proxy Operator](docs/developing/connect_rpc.md), however, the transaction cost is covered by us. This means if you bring assets into the Neon network, then the first transactions are free.
+
+## How does it work?
+
+1. The user transfers tokens to a new Neon account
+
+> You may use NeonPass to move SPL tokens from Solana or bridge assets across with a partner's bridge
+
+2. The Proxy Operator nominates the Neon account as eligible for N gasless transactions.
+
+3. The user sets up a swap of the transferred tokens for NEON.
+
```

Comment:
We need to include something like this:
a. The DEX makes an RPC request to the proxy for the gas price.
b. If the account is eligible for gasless transactions, the gas price will be set to 0. Otherwise, it will be calculated in the usual way.
c. The returned value from the RPC request should be used as the gas price value during transaction creation.
Here is the code example: link to the code